demos/radio.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL Radio Demo</title>
+    <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+    <style type="text/css">
+    .surface {
+      box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
+      0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+      0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+
+      position: relative;
+    }
+     section {
+       margin: 48px;
+     }
+     body {
+       font-family: sans-serif;
+     }
+     button {
+       display: block;
+       margin: 16px 0;
+     }
+     .mdl-radio {
+       margin: 8px;
+     }
+     .mdl-radio span {
+       display: inline-block;
+       line-height: 24px;
+       margin-left: 4px;
+       vertical-align: top;
+     }
+    </style>
+  </head>
+  <body>
+    <main>
+      <section>
+        <label for="radio1" class="mdl-radio">
```

Comment:
hmm, it might make sense to have this DOM structure mirror that of `MDLCheckbox`, e.g. `mdl-radio-wrapper`, `mdl-radio-wrapper__layout`, etc.. WDYT? 

If we want to stick with wrapping everything in a label, we can add that work to be done to #4471